scripts/start-console.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 
 npm_package_consolePlugin_name="nmstate-console-plugin"
 
-CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
+CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:4.14"}
```

Comment:
Not sure what is the reason of this change.. Do we really want this?

Reply:
This is because on `:latest` we have to refresh everytime the browser data otherwise the plugin will not show up. Just like happens with kubevirt-plugin. In this case our e2e tests fails for this reason. So we can keep using 4.14 and when this issue is resolved, reverse this change.